utils/sb2-upgrade-config

```diff
@@ -21,56 +21,54 @@ SBOX_DIR=$(readlink -f $SBOX_SHARE_DIR/../..)
 SBOX_TARGET=$1
 SBOX_CONFIG_DIR=~/.scratchbox2/$SBOX_TARGET/sb2.config.d 
 
-function exit_error()
+exit_error()
 {
-	echo "$my_path: Error: $@"
+	echo "$my_path: Error: " "$@"
```

Comment:
Why this change?

Reply:
Because the way it expands. `$@` is an array which will expand into `$1`, `$2` etc,
which means each counts as it's own argument if this behavior is intended the string
and the following `$@` should be quoted separately or if they should be combined
into one string `$*` fits better.

I changed it to the latter now.